.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
```

Comment:
JSON doesn't normally support comments, and the syntax highlighting is getting really nasty here. Are you sure we can leave these comments in?

Reply:
They were present when I autogenerated that file.
I think VSCode uses JSON5 or similar that supports comments for those files, as regular json does not.
Prefer to get rid of them?

Reply:
If they were autogenerated, I say leave 'em, they'll be fine!